docker-compose.postgres-test.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  postgresql-test:
+    image: postgres:16
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: "intermediary-test"
+      POSTGRES_PASSWORD: "changeIT!" # pragma: allowlist secret
+      POSTGRES_USER: "intermediary"
+    ports:
+      - 5434:5432
```

Comment:
Is it supposed to be `5432:5434`?

Reply:
The non-test version is 5433:5432, so this is just incrementing the 5433 to match. I think the first number is the Docker host port (so what any services on your machine or a DB client or whatever would connect to) and the second is the container's port inside docker. Since the two DBs (`intermediary` and `intermediary-test`) are running on different containers, it's okay they're both using 5432 (the default postgres port), and then the 5434 is what the load tests connect to